framework/tests/whole-program-inference/non-annotated/Anonymous.java

```diff
@@ -31,37 +35,3 @@ void expectsParent(@Parent int t) {}
         return (@Sibling1 int) 0;
     }
 }
-
-class UsesAnonymous {
```

Comment:
I'm curious about why this was removed.

Reply:
It wasn't; it's now in its own file. The reason for separating it is that stub files don't support annotations on anonymous inner classes, so stub-based WPI fails this part of the test. In the `build.gradle` file, you'll find a comment explaining that along with the logic that skips the test.